dev/NavigationView/NavigationView_InteractionTests/CommonTests.cs

```diff
@@ -1675,6 +1675,94 @@ public void VerifyNoCrashWhenSwitchingPaneDisplayModeWithAutoWrappedElements()
             }
         }
 
+        [TestMethod]
+        public void VerifyNavigationViewItemChildrenFlyoutMenuCornerRadius()
+        {
+            using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", "HierarchicalNavigationView Markup Test" }))
+            {
+                Log.Comment("Set PaneDisplayMode to LeftCompact.");
+                var panelDisplayModeComboBox = new ComboBox(FindElement.ByName("PaneDisplayModeCombobox"));
+                panelDisplayModeComboBox.SelectItemByName("LeftCompact");
+                Wait.ForIdle();
+
+                VerifyChildrenFlyoutMenuCornerRadius();
+
+                // Refresh the cache to make sure that the flyout object we are going to be searching for
+                // does not return as a false positive due to the caching mechanism.
+                ElementCache.Clear();
+
+                Log.Comment("Set PaneDisplayMode to Top.");
+                panelDisplayModeComboBox.SelectItemByName("Top");
+                Wait.ForIdle();
+
+                VerifyChildrenFlyoutMenuCornerRadius();
+
+                void VerifyChildrenFlyoutMenuCornerRadius()
+                {
+                    Log.Comment("Verify that the children menu flyout used in this test is closed.");
+                    var childItem = FindElement.ByName("Menu Item 2");
+                    Verify.IsNull(childItem, "Menu Item 1's children menu flyout should have been closed.");
+
+                    Log.Comment("Select Menu Item 1 which should open children flyout.");
+                    var item = FindElement.ByName("Menu Item 1");
+                    InputHelper.LeftClick(item);
+                    Wait.ForIdle();
+
+                    childItem = FindElement.ByName("Menu Item 2");
+                    Verify.IsNotNull(childItem, "Menu Item 1's children menu flyout should have been open.");
+
+                    Log.Comment("Get CornerRadius of Menu Item 1's children menu flyout.");
+                    FindElement.ByName<Button>("GetMenuItem1ChildrenFlyoutCornerRadiusButton").Invoke();
+
+                    // A CornerRadius of (4,4,4,4) is the current value for flyouts.
+                    TextBlock menuItem1ChildrenFlyoutCornerRadiusTextBlock = new TextBlock(FindElement.ByName("MenuItem1ChildrenFlyoutCornerRadiusTextBlock"));
+                    Verify.AreEqual("4,4,4,4", menuItem1ChildrenFlyoutCornerRadiusTextBlock.DocumentText);
+                }
+            }
+        }
+
+        [TestMethod]
+        public void VerifyOverflowMenuCornerRadius()
```

Comment:
There is a `VerifyMoreButtonIsOnlyReadOnce` test in this class for the overflow button: https://github.com/microsoft/microsoft-ui-xaml/blob/11ccea76a9bd12d3ee74f7481abd0690de5607e1/dev/NavigationView/NavigationView_InteractionTests/CommonTests.cs#L1500

We should probably move away from naming the overflow button in Top mode the "More" button as its really the overflow menu button and the "More" default button label will be dropped with PR https://github.com/microsoft/microsoft-ui-xaml/pull/3063

Proposed new name as part this PR: VerifyOverflowButtonIsOnlyReadOnce

Reply:
To users, the button will still show "See more" on hover, so it didn't really change it's appearance to the end user beside the label not showing anymore but rather being a tooltip. So naming it "OverFlowButton" isn't really fitting (what is an overflow button? A button that flows over?). Maybe `VerifyShowOverFlowButtonIsOnlyReadOnce`?

Reply:
WinUI uses the term "OverflowButton" internally and in public APIs in quite a few places for the NavView:

https://github.com/microsoft/microsoft-ui-xaml/blob/9b5509d02d1c8d074dbe6b0705d5893bfa87a7d4/dev/NavigationView/NavigationView.xaml#L271

And for its resources, like [TopNavigationViewOverflowButtonMargin](https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/NavigationView/NavigationView_rs1_themeresources.xaml#L255) or public styles like `NavigationViewOverflowButtonStyleWhenPaneOnTop` and `NavigationViewOverflowButtonNoLabelStyleWhenPaneOnTop`.

That said, the CommandBar names its overflow menu button "MoreButton" as well: https://github.com/microsoft/microsoft-ui-xaml/blob/9b5509d02d1c8d074dbe6b0705d5893bfa87a7d4/dev/CommonStyles/CommandBar_themeresources.xaml#L846

And as can be seen in that snippet, its overflow menu button tresources seem to use that term as well: `CommandBarMoreButtonMargin` 

Poitn being: We should settle on a consistent term here I believe. I was thinking about naming it the "OverflowMenuButton" (a button which shows the overflow menu on click) instead of "OverflowButton" but for now both "MoreButton" and "OverflowButton" are used in WinUI.

Reply:
@YuliKl We should pick a way to refer to this button and be consistent. Referring to the component in a consistent way makes searching the OSS code for the implementation or googling from the source code.

Reply:
I think OverflowButton is correct. CommandBar and NavigationView are arriving at the same visual representation of a "..." button from different places. The original purpose of CommandBar's More button was to 1) make the CommandBar taller so the button labels would become visible, and 2) show a flyout with secondary commands. NavigationView's Overflow button, on the other hand, was added to hold the overflow - primary commands that don't fit in the top NavigationView due to physical size. If space allows, these items move out of the overflow into the main area.